db/migrate/20191007140446_add_user_accuracy_bonus_to_reviewable_scores.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class AddUserAccuracyBonusToReviewableScores < ActiveRecord::Migration[6.0]
+  def up
+    # Add user_accuracy_bonus to reviewable_scores
+    execute <<~SQL
+    ALTER TABLE reviewable_scores
```

Comment:
The ALTER part should use rails helpers to be consistent with our other migrations.